### PR TITLE
Prevent overflow of timeline titles

### PR DESCRIPTION
**Pull request by n0izn0iz**
https://bluenove.atlassian.net/browse/DEVAS-991

### CODE REVIEW

File: assembl/static2/css/components/timeline.scss [hunk 1]
```diff
@@ -66,32 +73,45 @@
...
+
+    .timeline-bars {
+      display: inline-block;
+      width: calc(100% - 10px);
+      height: auto;
+      position: absolute;
+      top: 9px;
+      left: 12px;
+
+      .timeline-bar-1 {
```
Comment (ayyazdaniaryan) [inline assembl/static2/css/components/timeline.scss]: Sorry mate, the name `timeline-bar-1` is not very clear, without having to dig deep into the DOM code. Readability is important. Which part of the timeline does this represent, top/bottom, left/right, etc?

File: assembl/static2/css/components/timeline.scss [hunk 2]
```diff
@@ -66,32 +73,45 @@
...
-    z-index: 0;
-  }
-
-  .timeline-bar-2 {
-    background-color: $first-color;
-    height: 6px;
-    margin: 6px 0 0 11px;
-    position: absolute;
-    z-index: 1;
+      .timeline-bar-2 {
```
Comment (ayyazdaniaryan) [inline assembl/static2/css/components/timeline.scss]: Likewise...

File: assembl/static2/css/components/timeline.scss [hunk 3]
```diff
@@ -38,6 +38,12 @@
 }
 
 .minimized-timeline {
+  height: 100%;
+  position: relative;
+  flex-grow: 1;
+  flex-basis: 0;
```
Comment (paolina92) [inline assembl/static2/css/components/timeline.scss]: Curious to see the result. Did you check the browsers compatibility? I think so...

Comment (n0izn0iz) [reply assembl/static2/css/components/timeline.scss]: works on all browsers

**Pull request merged**